x-pack/plugins/secops/public/components/timeline/body/renderers/__snapshots__/get_column_renderer.test.tsx.snap

```diff
@@ -2,23 +2,6 @@
 
 exports[`get_column_renderer renders correctly against snapshot 1`] = `
 <span>
-  <Connect(DraggableWrapperComponent)
-    dataProvider={
-      Object {
-        "and": Array [],
-        "enabled": true,
-        "excluded": false,
-        "id": "id-timeline-column-event_id-for-event-1",
-        "kqlQuery": "",
-        "name": "event.id: 1",
-        "queryMatch": Object {
-          "field": "event.severity",
-          "value": 1,
-        },
-      }
-    }
-    key="timeline-draggable-column-event.id-for-event-1"
-    render={[Function]}
-  />
+  --
```

Comment:
These should be happy path based data to showing a column renderer rendering and if we see a `--` then that indicates an issue with either the mock file or the unit test. See comments of mine below but this snapshot did help me see that we have something wrong with a unit test.